<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>600</height>
    </size>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>551</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1001</width>
       <height>551</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="currentIndex">
      <number>6</number>
     </property>
     <widget class="QWidget" name="page_1">
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1001</width>
         <height>551</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>20</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>0</y>
          <width>20</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-1</y>
          <width>991</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>961</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>image/1st complete.jpg</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>961</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 255, 255);
background-color: rgb(12, 32, 53);
text-align: center</string>
        </property>
        <property name="text">
         <string>FOR AUTOMATION IT'S NEEDED.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1001</width>
         <height>551</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>1</y>
          <width>20</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>1</y>
          <width>20</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>991</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>961</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>image/2nd complete.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>label_13</zorder>
       <zorder>label_8</zorder>
       <zorder>label_9</zorder>
       <zorder>label_10</zorder>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1001</width>
         <height>551</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>1</y>
          <width>20</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1001</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>20</y>
          <width>21</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>981</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>image/3rd complete.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>label_24</zorder>
       <zorder>label_14</zorder>
       <zorder>label_15</zorder>
       <zorder>label_25</zorder>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QFrame" name="frame_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1001</width>
         <height>551</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>20</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1011</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>20</y>
          <width>21</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_30">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>961</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>image/weblinker.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_32">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>961</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 255, 255);
background-color: rgb(12, 32, 53);
text-align: center</string>
        </property>
        <property name="text">
         <string>HOW IT WORKES..</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <zorder>label_30</zorder>
       <zorder>label_16</zorder>
       <zorder>label_17</zorder>
       <zorder>label_26</zorder>
       <zorder>label_32</zorder>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <widget class="QFrame" name="frame_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1011</width>
         <height>551</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>1</y>
          <width>20</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>991</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_27">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>10</y>
          <width>21</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_33">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>961</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>image/timetable manager.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>label_33</zorder>
       <zorder>label_18</zorder>
       <zorder>label_19</zorder>
       <zorder>label_27</zorder>
      </widget>
     </widget>
     <widget class="QWidget" name="page_6">
      <widget class="QFrame" name="frame_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1001</width>
         <height>551</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>1</y>
          <width>20</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1001</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_28">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>20</y>
          <width>21</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_31">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>961</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>image/lecname understading....jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>label_31</zorder>
       <zorder>label_20</zorder>
       <zorder>label_21</zorder>
       <zorder>label_28</zorder>
      </widget>
     </widget>
     <widget class="QWidget" name="page_7">
      <widget class="QFrame" name="frame_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1001</width>
         <height>551</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>1</y>
          <width>20</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1001</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>20</y>
          <width>21</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 37, 39);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_35">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>961</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>image/antaryamimi.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>label_35</zorder>
       <zorder>label_22</zorder>
       <zorder>label_23</zorder>
       <zorder>label_29</zorder>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>549</y>
      <width>1001</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>5</y>
       <width>1001</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(33, 37, 39);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="next">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>221</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
	color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 255, 255);
}
QPushButton:hover {
    background-color: rgb(103, 255, 2);
}
QPushButton:pressed {
    background-color: rgb(224, 0, 0);
    border-style: inset;
}
</string>
     </property>
     <property name="text">
      <string>NEXT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="prev">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>221</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
	color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 255, 255);
}
QPushButton:hover {
    background-color: rgb(103, 255, 2);
}
QPushButton:pressed {
    background-color: rgb(224, 0, 0);
    border-style: inset;
}
</string>
     </property>
     <property name="text">
      <string>PREVIOUS</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
